package.json

```diff
@@ -16,13 +17,52 @@
     "pre-commit": "pre-commit run --all-files",
     "prepare": "npm run clean && npm run build",
     "prettier": "prettier --write .",
-    "rollup": "rollup -c",
+    "rollup": "wireit",
+    "rollup:watch": "rollup -c --watch",
     "server": "npm run build && npm run server-no-build --",
     "server-no-build": "node lib/cjs/bidiServer/index.js",
-    "test": "npm run build && mocha",
-    "watch": "tsc -b src/tsconfig.json --watch && rollup -c --watch",
+    "test": "wireit",
+    "test:watch": "npm run build && mocha --watch",
```

Comment:
This can be a wireit command as well dependency of build and command `mocha --watch`

Reply:
Done